### PR TITLE
Add support info for worker versioning

### PR DESCRIPTION
**Pull request by lorensr**
Supercedes #2127

### CODE REVIEW

**Comment by lorensr** [comment]
image

**Review by Sushisource — APPROVED** [review]

**Review by flossypurse — APPROVED** [review]

File: docs-src/concepts/what-is-worker-versioning.md [hunk 1]
```diff
@@ -7,9 +7,10 @@ tags:
   - explanation
   - versioning
 ssdi:
-  - Introduced in Temporal Server version 1.21.0
+  - Introduced in Temporal Server version [1.21.0](https://github.com/temporalio/temporal/releases/tag/v1.21.0)
+  - Available in Go SDK version [1.23.0](https://github.com/temporalio/sdk-go/releases/tag/v1.23.0)
+  - Available in Java SDK version [1.20.0](https://github.com/temporalio/sdk-java/releases/tag/v1.20.0)
   - Will come to CLI in version 0.10.0
-  - Coming soon to SDKs
```
Comment (Sushisource) [inline docs-src/concepts/what-is-worker-versioning.md]: Maybe keep this and add "remaining SDKs"

**Review by Sushisource — APPROVED** [review]

File: docs-src/concepts/what-is-worker-versioning.md [hunk 2]
```diff
@@ -6,6 +6,12 @@ description: Worker Versioning lets you more easily deploy changes to Workflow D
 tags:
   - explanation
   - versioning
+ssdi:
+  - Introduced in Temporal Server version [1.21.0](https://github.com/temporalio/temporal/releases/tag/v1.21.0)
+  - Available in [Go SDK](/dev-guide/golang/versioning#worker-versioning) version [1.23.0](https://github.com/temporalio/sdk-go/releases/tag/v1.23.0)
+  - Available in [Java SDK](/dev-guide/java/versioning#worker-versioning) version [1.20.0](https://github.com/temporalio/sdk-java/releases/tag/v1.20.0)
+  - Will come to CLI in version 0.10.0
+  - Not yet available in Cloud
```
Comment (flossypurse) [inline docs-src/concepts/what-is-worker-versioning.md:14]:
```suggestion
  - Not yet available in Temporal Cloud
```

**Review by flossypurse — APPROVED** [review]
I am not 100% sure about the statements about what is being released where in the future - but am sure we can err on the side of good intent and let this roll.

File: docs-src/concepts/what-is-worker-versioning.md (hunk 1 above)
Comment (lorensr) [reply docs-src/concepts/what-is-worker-versioning.md]: Until we hear back from product on messaging, I'm uncertain about saying "coming soon". Also, we don't say anything about other SDKs under Update, or about PHP under Schedules

Comment (lorensr) [reply docs-src/concepts/what-is-worker-versioning.md]: More info: https://temporaltechnologies.slack.com/archives/C01H1G7J98F/p1687789090371359?thread_ts=1687761828.187439&cid=C01H1G7J98F

**Pull request merged**